bin/tblextr.py

```diff
@@ -59,6 +60,48 @@
 var_table = {}
 #############################################################
 
+def json_metadata_gen(sysinfo_file, index):
```

Comment:
Shold be just metadata_gen()
Should consume output file and cmdline to generate info.
Should append the info to the output file.
Example to use it:
metadata_gen(sysinfo_file, '/opt/rocm/bin/rocminfo')

Reply:
Done